benchmark-android/build.gradle

```diff
@@ -0,0 +1,61 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.3.3' // jcenter has the latest
+    }
+}
+
+description = 'Conscrypt: Android Benchmarks'
+
+ext {
+    androidHome = "$System.env.ANDROID_HOME"
+    androidSdkInstalled = file("$androidHome").exists()
+    androidVersionCode = 1
+    androidVersionName = "$version"
+    androidMinSdkVersion = 19
+    androidTargetSdkVersion = 25
+    androidBuildToolsVersion = "25.0.0"
+}
+
+if (androidSdkInstalled) {
+    apply plugin: 'com.android.library'
+
+    android {
+        compileSdkVersion androidTargetSdkVersion
+        buildToolsVersion androidBuildToolsVersion
+
+        compileOptions {
+            sourceCompatibility androidMinJavaVersion;
+            targetCompatibility androidMinJavaVersion
+        }
+
+        defaultConfig {
+            minSdkVersion androidMinSdkVersion
+            targetSdkVersion androidTargetSdkVersion
+            versionCode androidVersionCode
+            versionName androidVersionName
+        }
+        sourceSets.main {
+            java {
+                srcDirs = [
+                        "${rootDir}/benchmark-base/src/main/java",
```

Comment:
I removed the benchmark-base subproject, so benchmark-android and benchmark-openjdk both include its source directly.  This is the same pattern used with common/, it doesn't get compiled directly but android/ and openjdk/ include its source.  It makes it a bit easier because there are circular dependencies between the platform-specific bits and the platform-agnostic bits, so this just lets them get compiled together.